tests/acceptance/guid-node/registrations-test.ts

```diff
@@ -57,10 +61,95 @@ module('Acceptance | guid-node/registrations', hooks => {
         assert.dom('[data-test-node-card-heading]').includesText(title);
     });
 
+    test('logged in read-permission, 1 registration, 1 draft', async assert => {
+        server.create('user', 'loggedIn');
+        const node = server.create('node', { id: 'decaf', currentUserPermissions: [Permission.Read] });
+        server.create('registration', {
+            registeredFrom: node,
+        });
+        server.create('draft-registration', {
+            branchedFrom: node,
+        }, 'currentUserIsReadOnly');
+
+        const url = `/${node.id}/registrations`;
+
+        await visit(url);
+
+        assert.equal(currentURL(), url, `We are on ${url}`);
+
+        assert.dom('[data-test-new-registration-button]').doesNotExist(
+            'read-permission contributors cannot start new registrations',
+        );
+
+        assert.dom('[data-test-node-card]').exists({ count: 1 }, 'One registration card shown');
+
+        assert.dom('[data-test-registrations-container] a[href="#drafts"]').exists(
+            'read-permission contributors can see drafts tab',
+        );
+
+        await untrackedClick('[data-test-registrations-container] a[href="#drafts"]');
```

Comment:
There's a data-analytics-scope on https://github.com/CenterForOpenScience/ember-osf-web/pull/1174/files#diff-37a43077c37dd95fd2ec97262b1cc50a06dc8ae3f0283e36bde01f093d9e8bc1R50 does that not send tracking?

Reply:
There's definitely tracking associated when clicking this pane, but the problem I see when using `click` from `ember-osf-web/tests/helpers` (the one that checks for analytics), is that `click` helper checks if the element has a `data-analytics-name` attribute (which the element indeed does not have the `data-analytics-name` attr). Instead the analytics for this is being fired when the action tied to `<BsTab @changeTab>` is called.